### DISCUSSION TITLE
Remote start of heating havc, charging plans or parking location?

### DISCUSSION THREAD

**Discussion by leowinterde** [pypolestar/polestar_api#86] [Q&A]
Hey,
is it possible to start the heating havc or activate any of the charging plans via the api - just like in the app? Or to read out the parking location?

**Comment by 3shirts** [comment]
I'd love to see this too, particularly the heating and the parking location.

**Comment by 3shirts** [comment]
Just to add to this: Is there any way to set a precondition timer like you can in the app? It would be great if I could automate that based on my Ohme home charger setting.
Even if you could just read the existing timers, I could do it the other way round (set it in the Polestar app and automate on Ohme)

**Comment by motionist** [comment]
I'd like to see both the next heating start (or alternatively heating schedules in general) and the location of the vehicle in order to determine if I should allow the charger at home to charge the car. If the car is @home (condition) and connected (condition) and heating scheduled (trigger), then charger should be turned on.

**Reply by 3shirts** [reply]
Do you use [Car Stats Viewer](https://github.com/Ixam97/CarStatsViewer)? It's an app for the car which gives you lots of detail and can use a webhook to communicate with Home Assistant. It can send location data. Unfortunately it's not approved for the Play Store yet so you have to join one of the dev's test tracks but it's easy enough to do and works well. Downside is that it only works while the app is running so not much use when parked or if a different user (who doesn't have the app) is driving.

**Reply by leowinterde** [reply]
Sure CarStatsViewer is awesome, but the Location Information should also be available from the Polestar Api

**Reply by motionist** [reply]
The challenge with CSV is that it requires all users to use it (or at least have it installed).
